src/lib/schematics/update/tests/class-names/expected.fixture

```diff
@@ -0,0 +1,14 @@
+import {CdkConnectedOverlay, CdkOverlayOrigin} from '@angular/cdk/overlay';
```

Comment:
I don't think "fixture" is the right word for this. I'd also like to keep the `.ts` extension so that the files get the right syntax highlighting everywhere. 

How about 
```
schematics/update/test-cases/ts-class-name.before.ts
schematics/update/test-cases/ts-class-name.after.ts
```

Other possible pairs could be "input" / "output"

cc @hansl in case he has any ideas for this

Reply:
`input` and `output` sounds better IMO. I'd even go with `input` and `expected_output`.

In regards to the extension. Technically it's not real valid TypeScript code because the imports are not valid, and just to mock the Material and CDK packages. I don't really care that much though.